addons/explosives/functions/fnc_daisyChainTo.sqf

```diff
@@ -0,0 +1,42 @@
+/*
+ * Author: Loren Luke Leimer
+ * Daisy Chains two explosives together
+ *
+ * Arguments:
+ * 0: Explosive <OBJECT>
+ * 1: Target Explosive <OBJECT>
+
+ *
+ * Return Value:
+ * <NOTHING>
+ *
+ * Example:
+ * [_explosive,_target] call ACE_Explosives_fnc_daisyChainTo;
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+  
+private ["_unit","_target"];
+
+_explosive = _this select 0;
+_target = _this select 1;
+
+if (isNil QGVAR(DaisyChains)) then {
+    GVAR(DaisyChains) = [];
+};
+
+_result = false;
+
+{
+    if (count GVAR(DaisyChains) < 1) exitWith {};
+    if (_x select 0 == _explosive) then {
+	    if (_x select 1 == _target) then {_result = true};
+	};
+} forEach GVAR(DaisyChains);
```

Comment:
You could just do this in the forEach body:

`if (_x isEqualTo [_explosive, _target]) exitWith { _result = true; };`